rasa_core/broker.py

```diff
@@ -132,3 +136,62 @@ def publish(self, event: Dict) -> None:
 
         self.event_logger.info(json.dumps(event))
         self.event_logger.handlers[0].flush()
+
+
+class KafkaProducer(EventChannel):
+    def __init__(self, host, sasl_plain_username=None,
```

Comment:
mhm why not just `sasl_username`? (or is it called `sasl_plain_username` in kafka?)

Reply:
That's because it's called `sasl_plain_protocol` in `kafka-python` lib. But you're right, it's clear by the security_protocol.